caesar_cipher/lib/cipher.rb

```diff
@@ -0,0 +1,29 @@
+class Cipher
+
+  def initialize(rot_key, raw_message)
+    @rot_key = rot_key
+    @raw_message = raw_message
+  end
+
+  def encrypt
```

Comment:
@zzokki81 this doesn't allow us to do what I've asked in this comment: https://github.com/zzokki81/Ruby/pull/1#discussion_r450650835